### PR TITLE
Update to v0.43.4

### PR DESCRIPTION
**Pull request by archerallstars**

### CODE REVIEW

File: io.anytype.anytype.metainfo.xml [hunk 1]
```diff
@@ -25,7 +25,7 @@
   <launchable type="desktop-id">io.anytype.anytype.desktop</launchable>
 
   <releases>
-    <release version="0.43.1" date="2024-10-24" />
+    <release version="0.43.4" date="2024-10-31" />
```
Comment (Adda0) [inline io.anytype.anytype.metainfo.xml:28]: The new release should have been added as a new record, not replacing the last one.

Comment (Adda0) [inline io.anytype.anytype.metainfo.xml:28]:
```suggestion
    <release version="0.43.4" date="2024-10-31" />
    <release version="0.43.1" date="2024-10-24" />
```
Update to 0.43.4

The new release should have been added as a new record, not replacing the last one.

**Comment by mhymny** [comment]
https://status.flathub.org/issues/2024-11-03-repo-exp/

**Comment by mhymny** [comment]
bot, build io.anytype.anytype

**Pull request merged**